src/main/java/com/twilio/oai/common/CsharpResolver.java

```diff
@@ -264,4 +293,41 @@ public List<CodegenParameter> resolveParameter(List<CodegenParameter> parameters
         }
         return parameters;
     }
+
+    private String titleCase(String input){
+        input.replace(" ","");
+        String[] split = input.split("_");
+        String result = "";
+        for(int i = 0; i < split.length; i++){
+            String item = split[i];
+            result+= Character.toTitleCase(item.charAt(0)) + item.substring(1);
+        }
+        return result;
+    }
+
+    private void extractEnums() {
+        Map<String, Object> usingMappings = conventionMap.get("library");
+        usingMappings.forEach((key, value) -> {
+            if(value != null){
+                if(value instanceof String && ((String) value).equalsIgnoreCase("Twilio.Types")){
+                    enumsDict.add(titleCase(key));
+                } else if (value instanceof ArrayList) {
+                    ((ArrayList<String>) value).forEach(item -> {
+                        if(item.equalsIgnoreCase("Twilio.Types")){
+                            enumsDict.add(titleCase(key));
```

Comment:
Is this use case covers if in "library",   "Twilio.Types" is present in array ?
Example for prefixed_collapsible_map: ["Twilio.Converters", "System.Collections.Generic"]

If yes, I don't think it will be the use case for enums because it is used for importing in options file where 
example from getparams method:
p.AddRange(PrefixedCollapsibleMap.Serialize(AddOnsData, "AddOns"));
AddRange --> present in "Twilio.Converters"
PrefixedCollapsibleMap --> present in "System.Collections.Generic"

What are your thoughts ?

Reply:
Yes this covers when Twilio.Types is present in an array. Twilio.Types will be present if any of the Twilio Types / enums are used. In such cases we will be including this